scripts/scil_bundle_fixel_analysis.py

```diff
@@ -0,0 +1,291 @@
+                   help='List of the names of the bundles, in the same order '
+                        'as they were given. \nIf this argument is not used, '
+                        'the script assumes that the name of the bundle \nis '
+                        'its filename without extensions.')
+
+    p.add_argument('--dps_key', default=None, type=str,
+                   help='Key to access the data per streamline to use as '
+                        'weight when computing the maps, \ninstead of the '
+                        'number of streamlines. [%(default)s].')
+
+    g = p.add_argument_group(title='Mask parameters')
+
+    g.add_argument('--abs_thr', default=0, type=float,
+                   help='Value of density maps threshold to obtain density '
+                        'masks, in number of streamlines \nor streamline '
+                        'weighting if --dps_key is given. Any number of '
+                        'streamlines \nor weight above this value will '
+                        'pass the absolute threshold test [%(default)s].')
+
+    g.add_argument('--rel_thr', default=0.01, type=float,
+                   help='Value of density maps threshold to obtain density '
+                        'masks, as a ratio of the normalized density '
+                        '\nAny normalized density above '
+                        'this value will pass the relative threshold test. '
+                        '\nMust be between 0 and 1 [%(default)s].')
+
+    g.add_argument('--norm', default="voxel", choices=["fixel", "voxel"],
+                   help='Way of normalizing the density maps. If fixel, '
+                        'will normalize the maps per fixel, \nin each voxel. '
+                        'If voxel, will normalize the maps per voxel. '
+                        '[%(default)s]')
+
+    p.add_argument('--max_theta', default=45,
+                   help='Maximum angle between streamline and peak to be '
+                        'associated [%(default)s].')
+
+    p.add_argument('--split_bundles', action='store_true',
+                   help='If set, save the density maps for each bundle '
+                        'separately \nin addition to the all in one version.')
+
+    p.add_argument('--split_fixels', action='store_true',
+                   help='If set, save the density maps for each fixel '
+                        'separately \nin addition to the all in one version.')
+
+    p.add_argument('--single_bundle', action='store_true',
+                   help='If set, will save the single-fiber single-bundle '
+                        'masks as well. \nThese are obtained by '
+                        'selecting the voxels where only one bundle is '
+                        'present \n(and one fiber/fixel).')
+
+    p.add_argument('--bundles_mask', action='store_true',
+                   help='If set, save the bundle mask for each bundle.')
+
+    add_overwrite_arg(p)
+    add_processes_arg(p)
+    add_verbose_arg(p)
```

Comment:
I feel we should add a --out_dir option to this script, no? 

What if the user wants to rename the output filenames? Maybe something "a la" compute_DTI script?

Reply:
I would rather have three new options: 
1) --out_dir option to say where to put all the output files; 
2) --prefix option to ADD a prefix to all the output files, for example, the subject number or something like that.
3) --suffix option to ADD a suffix to all the output files, for example, the subject number or something like that.

The way scil_dti_metrics.py works would change the whole --split_bundles, --split_fixels and --single_bundle thing.

Reply:
Ok good call.